tests/globalhelper/helper.go

```diff
@@ -264,3 +270,134 @@ func AppendContainersToDeployment(deployment *v1.Deployment, containersNum int,
 
 	return deployment
 }
+
+// CreateAndWaitUntilReplicaSetIsReady creates replicaSet and wait until all replicas are up and running.
+func CreateAndWaitUntilStatefulSetIsReady(statefulSet *v1.StatefulSet, timeout time.Duration) error {
```

Comment:
Do we need this func on global level? If it's not used in other packages, please move it into package's level